### ISSUE TITLE
[dev] Average burst damage does not get affected by Winds of Fate

### ISSUE THREAD

**Issue by Lilylicious** [PathOfBuildingCommunity/PathOfBuilding#5471]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### How is Path of Building expected to behave?

Burst DPS should be affected by Winds of Fate when there are multiple hits.

### How does Path of Building behave?

Average burst damage is not influenced by Winds of Fate whatsoever.

### Character build code

```shell
https://pobb.in/MCD6Rlt0HAig
```

**Comment by Regisle** [comment]
![image](https://user-images.githubusercontent.com/49933620/209588033-c864941b-363c-472f-abf2-0bca11d142ea.png)
unable to reproduce, it seems correct to me?

though why do you have lucky crits enabled on your build?

**Comment by QuickStick123** [comment]
I agree it looks fine so I am probably missing something lucky crit are properly used to minimise downside of weapons 50% crit chance cap.
![image](https://user-images.githubusercontent.com/31533893/209588398-c40e92a2-b7a6-4239-937e-8e8efa8f1b25.png)

**Comment by Regisle** [comment]
Ahh I missed where it was coming from, wondered why focus was an option but unticked, forgot it rolls on belt

**Comment by Lilylicious** [comment]
You are correct, it looks correct. I got confused on this one.

**Comment by Lilylicious** [comment]
I'm not a huge fan of how the damage is displayed, even though it's correct. For one, it's not correct for ailments.

With 75% crit chance, we do 0 damage with non-crits and 100% damage with crits, which in PoB results in a hit that deals 75% of our critical strike damage. That means for the purposes of estimating shock/chill/freeze, we're doing 75% damage when it should be 100%.

More correct behaviour would be for average hit to be unaffected, while DPS or hit rate is affected. For burst damage, we'd change the amount of burst hits, not the average hit damage.

**Comment by QuickStick123** [comment]
This is a more general problem and effects all sources of crit currently.
